model/model.js

```diff
@@ -60,10 +60,47 @@ async function insertCommentOnArticle(articleId, author, body) {
   return query.rows[0];
 }
 
+async function updateArticle(votesNum, articleId) {
+  if (!votesNum || !articleId) {
```

Comment:
We shouldn't need to handle this error with a custom reject. As a general rule always let psql throw an error if it's going to and only explore custom error handling if it doesn't.